tests/data/test358

```diff
@@ -0,0 +1,94 @@
+<testcase>
+<info>
+<keywords>
+HTTP
+HTTP GET
+Alt-Svc
+HTTP/2
+</keywords>
+</info>
+
+#
+# Server-side
+<reply>
+<data nocheck="yes">
+HTTP/1.1 200 OK
+Date: Thu, 09 Nov 2010 14:49:00 GMT
+Content-Length: 6
+Connection: close
+Content-Type: text/html
+Funny-head: yesyes
+Alt-Svc: h2=":9014", ma=315360000; persist=0
```

Comment:
Port 9014 is not guaranteed to run the HTTP/2 server. Use the `%HTTP2PORT` variable here instead! (works since d009bc2e56 landed) Should apply for all appearances of 9014 in these test cases. You can probably verify that it works by using `runtests.pl -b 9000` to move the base port.

This is also the reason why the alt-svc CI tests failed on travis just now.

Reply:
yes, though I initially found var substitution did not work in data section .. let me try again.